oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -694,11 +669,7 @@ && maxAgeExpired(
                         newSessionId);
 
             } else {
-                var previousSession = existingSession.get();
                 var previousSessionId = previousSessionIdFromCookie.get();
-                session =
-                        sessionService.updateWithNewSessionId(
```

Comment:
Does this mean we can't merge this PR until Auth have stopped reading the shared session entirely?